dbt/adapters/bigquery.py

```diff
@@ -179,6 +180,13 @@ def execute_model(cls, profile, model):
         model_name = model.get('name')
         model_sql = cls.format_sql_for_bigquery(model.get('injected_sql'))
 
+        materialization = dbt.utils.get_materialization(model)
+        allowed_materializations = ['view', 'ephemeral']
```

Comment:
this is a fine stopgap for 0.8.3. but this has me thinking about how to support this for extended materializations. i think that the bulk of this code will go away in the next version.